browser/ui/webui/brave_adblock_ui.h

```diff
@@ -7,12 +7,20 @@
 
 #include "brave/browser/ui/webui/basic_ui.h"
 
+class PrefChangeRegistrar;
+
 class BraveAdblockUI : public BasicUI {
  public:
   explicit BraveAdblockUI(content::WebUI* web_ui, const std::string& host);
   ~BraveAdblockUI() override;
 
  private:
+  void CustomizeWebUIProperties();
+  void RenderFrameCreated(content::RenderFrameHost* render_frame_host) override;
+  void OnPreferenceChanged();
+
+  std::unique_ptr<PrefChangeRegistrar> pref_change_registrar_;
```

Comment:
ditto

Reply:
I'm going to keep this as `std::unique_ptr` because then it can be forward declared. I see a lot of instances in chromium source in both ways. I will remove the destructor explicit reset though which I agree isn't needed.

Reply:
Agreed